### PR TITLE
Add prometheus tests

### PR DESCRIPTION
**Pull request by brian-avery**
This adds some initial prometheus tests for us...

-- Make build
-- Make test

Both using Prometheus built-in content

### CODE REVIEW

File: docker/maistra-builder_2.1.Dockerfile [hunk 1]
```diff
@@ -32,6 +32,7 @@ ENV LICENSEE_VERSION=9.11.0
 ENV GOLANG_PROTOBUF_VERSION=v1.3.2
 ENV FPM_VERSION=1.11.0
 ENV SHELLCHECK_VERSION=v0.7.1
+ENV PROMU_VERSION=0.7.0
```
Comment (jwendell) [inline docker/maistra-builder_2.1.Dockerfile:35]: I'm guessing this is the version that's supposed to work with the one we have in 2.1 branch

File: prow/config.gen.yaml [hunk 2]
```diff
@@ -1175,7 +1175,49 @@ presubmits:
...
+    branches:
+      - maistra-2.1
+    max_concurrency: 2
+    spec:
+      containers:
+      - image: "quay.io/maistra-dev/maistra-builder:2.1"
+        imagePullPolicy: Always
+        command:
+        - make
+        - test
```
Comment (jwendell) [inline prow/config.gen.yaml:1190]: Put `build` here (`make build test`) and delete the `prometheus-build` job.

File: docker/maistra-builder_2.1.Dockerfile [hunk 3]
```diff
@@ -148,6 +149,15 @@ RUN curl -sfLO https://github.com/protocolbuffers/protobuf/releases/download/v${
     mv bin/protoc /usr/local/bin && \
     rm -rf /root/*
 
+# Yarn
+RUN npm install --global yarn
+
+# Promu
+RUN curl -sfLO https://github.com/prometheus/promu/releases/download/v${PROMU_VERSION}/promu-${PROMU_VERSION}.linux-amd64.tar.gz && \
+    tar -zxvf promu-${PROMU_VERSION}.linux-amd64.tar.gz && \
+    mv promu-0.7.0.linux-amd64/promu /usr/local/bin  && \
```
Comment (jwendell) [inline docker/maistra-builder_2.1.Dockerfile:158]: `s/0.7.0/${PROMU_VERSION}/`

File: docker/maistra-builder_2.1.Dockerfile (hunk 1 above)
Comment (brian-avery) [reply docker/maistra-builder_2.1.Dockerfile:35]: Yeah, this is the one we're using for product.

File: prow/config.gen.yaml [hunk 4]
```diff
@@ -1175,7 +1175,29 @@ presubmits:
...
+    spec:
+      containers:
+      - image: "quay.io/maistra-dev/maistra-builder:2.1"
+        imagePullPolicy: Always
+        command:
+        - make
+        - build
+        - test
+        securityContext:
+          privileged: true
```
Comment (jwendell) [inline prow/config.gen.yaml:1193]: sorry, just saw this. Does it need this privilege?

Comment (brian-avery) [reply prow/config.gen.yaml]: I don't think so. Removed. Can always add it back if we need it.

**Review by jwendell — APPROVED** [review]

**Pull request merged**